scala-exercises-client/src/main/scala/config/Routes.scala

```diff
@@ -2,13 +2,9 @@ package config
 
 object Routes {
 
-  object Users {
-    val base = "/users"
-    def all = base + "/all"
-    def create = base + "/create"
-    def update(id: Long) = base + s"/update/$id"
-    def delete(id: Long) = base + s"/delete/$id"
-    def clear = base + "/clear"
+  object Exercises {
+    val base = "/exercises/sections"
+    def evaluate(section: String, category: String) = base + s"/$section/$category/evaluate"
```

Comment:
As far as I know it'd involve generating a [javascript router](https://www.playframework.com/documentation/2.3.0/ScalaJavascriptRouting) and using it from Scala.js. Unfortunately I haven't found a cleaner way to use it directly from client-side Scala. What's your opinion on this?